content/browser/renderer_host/render_process_host_impl.cc

```diff
@@ -4315,6 +4315,10 @@ bool RenderProcessHostImpl::MayReuseAndIsSuitable(
 bool RenderProcessHostImpl::MayReuseAndIsSuitable(
     RenderProcessHost* host,
     SiteInstanceImpl* site_instance) {
+  if (site_instance->RecordReplayForRecording()) {
+    return false;
+  }
```

Comment:
this is the most dogscience part of the whole thing - there are so many possible ways that chrome might decide to reuse a process.  This seems to help with my github tab test though.